src/plugins/curlget/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 if (DEPENDENCY_PHASE)
 	find_package(CURL)
-
 	if(NOT CURL_FOUND)
 		remove_plugin(curlget "Curl-dev not found")
 	endif ()
+	find_package(OpenSSL)
```

Comment:
```
 10         if(OPENSSL_FOUND AND CURL_FOUND)
 11             set (CURLGET_INCLUDE_DIRS ${CURL_INCLUDE_DIR} ${OPENSSL_INCLUDE_DIR})
 12             set (CURLGET_LIBRARY_DIRS ${CURL_LIBRARIES} ${OPENSSL_LIBRARIES})
 13         endif ()
...
 20         INCLUDE_DIRECTORIES
 21                 ${CURLGET_INCLUDE_DIRS}
 22         LINK_LIBRARIES
 23                 ${CURLGET_LIBRARY_DIRS}
```

anyway seems like it's a different issue. running "configure" in ccmake with all plugins yields the error mentioned before, but curlget and crypto_openssl + base64 are added to the plugins, and running "configure" again doesn't yield the error anymore + all plugins build file. 
i'm sure whats going on here though, because i'm still not familiar with cmake